src/content/en/updates/posts/2016/05/houdini-zh-cn-version.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: updates/post
+title: "Houdini – Demystifying CSS"
+description: "Houdini is a collection of APIs that expose the CSS engine’s internals to developers"
+published_on: 2016-05-20
+updated_on: 2016-05-20
+authors:
+  - surma
+tags:
+  - houdini
+  - css
+featured_image: /web/updates/images/2016/05/houdini/compworklet_small.png
+
+---
+
+
+你是否曾思考过(在页面布局中)CSS所承载的工作量？(紧接着)你修改了某个CSS属性的属性值，然后你整个网站瞬间以不同的layout展示在我们面前。CSS在页面布局方面确实有点魔力。(谁能告诉我，拥有这些魔力我们今后的道路将如何走?)到目前为止，身在web开发者社区的我们才能见证以及体验这些魔力。如果我们想提出自己的魔力该怎么办？如果我们想变成魔法师该怎么办？加入Houdini工作小组！  
+
+Houdini工作小组聚齐了来自Mozilla，Apple，Opera，Microsoft，HP，Intel以及Google工程师,工作小组的意图旨在把CSS引擎的部分模块暴露给web开发者。Houdini工作小组正在制订能够被W3C组织认可从而逐渐成为web标准的草案集。Houdini工作小组给他们自己定下了更高的目标，要将(由他们制订的)草案集变成规范草案，反过来，(由他们制订的)草案将会变成低层次且带有辅助性质的草案集。当有人谈论到"Houdini"时通常就表示他们谈论的是草案集。在我们写(自己筹备的)草案集的那段时间，草案列表都是处于未完成状态，甚至(草案列表中的)一些草案仅仅还是处于计划阶段。可见我们Houdini工作小组的准备工作是有多么的早啊。  
+
+![Houdini Compositor Worklet for transitions](http://7xl614.com1.z0.glb.clouddn.com/20160607095542.png)  
```

Comment:
Please use the original link to the video, not a screenshot that isn't linkable.